app/views/upload.scala.html

```diff
@@ -1,6 +1,47 @@
 @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
+  <div class="govuk-grid-row">
```

Comment:
Minor thing, but this file has a mix of 2 and 4 space indents. It should be 4 spaces everywhere.

@MancunianSam - I think you set up the linter. Do you know if we can apply it to Play templates so we don't have to worry about indentation ourselves? 🙂 

You can also configure IntelliJ to make the spacing consistent in Preferences->Editor->Code Style. I'm not sure whether it considers these files to be HTML, but you could try updating those settings.

Reply:
The linter is typescript only I'm afraid. There may be one for the templates, I haven't looked.

Reply:
Sorry, I meant Scalastyle rather than ESLint. (Is it fair to call Scalastyle a linter?)